include/sys/sysinfo.h

```diff
@@ -39,20 +39,24 @@
 
 struct sysinfo
 {
-  unsigned long uptime;
-  unsigned long loads[3];
-  unsigned long totalram;
-  unsigned long freeram;
-  unsigned long sharedram;
-  unsigned long bufferram;
-  unsigned long totalswap;
-  unsigned long freeswap;
-  unsigned short procs;
-  unsigned short pad;
-  unsigned long totalhigh;
-  unsigned long freehigh;
-  unsigned mem_unit;
-  char __reserved[256];
+  unsigned long uptime;    /* Seconds since boot */
+  unsigned long loads[3];  /* 1, 5, and 15 minute load averages */
+  unsigned long totalram;  /* Total usable main memory size */
+  unsigned long freeram;   /* Available memory size */
+  unsigned long sharedram; /* Amount of shared memory */
+  unsigned long bufferram; /* Memory used by buffers */
+  unsigned long totalswap; /* Total swap space size */
+  unsigned long freeswap;  /* Swap space still available */
+  unsigned short procs;    /* Number of current processes */
+  unsigned short pad;      /* Padding for alignment */
+  unsigned long totalhigh; /* Total high memory size */
+  unsigned long freehigh;  /* Available high memory size */
+  unsigned mem_unit;       /* Memory unit size in bytes */
+  char _f[64 - (
+    (12 * sizeof(unsigned long)) +
+    ( 2 * sizeof(unsigned short)) +
```

Comment:
I guess nxstyle won't accept alignment of numbers. Sorry :-/

```suggestion
    (2 * sizeof(unsigned short)) +
```